epsic_tools/mib2hdfConvert/E01/E01_widgets.py

```diff
@@ -0,0 +1,530 @@
+# code stared 07/11/2025 t 13:51:27 by ejr78941
+
+import re
+import numpy as np
+import time
+import os
+import glob
+import json
+from pathlib import Path
+import datetime
+from IPython.display import display, clear_output
+import ipywidgets.widgets as w
+import ipywidgets
+import matplotlib.pyplot as plt
+import h5py
+import traceback
+
+import subprocess
+
+
+
+#other functions:
+
+def delete_ptycho_flagging_files(basedir,year,session,subfolder,verbose=False):
+    '''This function is for deleting the autoptycho_is_done.txt files which prevent recontructions
+       from being run more than once'''
+
+    '''determine path path being searched and change directory to that path for glob'''
+    dest_path = f'/{basedir}/{year}/{session}/processing/Merlin/{subfolder}'
+    os.chdir(dest_path)
+    print('File being searched: ' + dest_path)
+
+
+def _create_ptyrex_bash_submit(json_files, script_folder, node_type, ptycho_time, verbose=False):
```

Comment:
When I try now, it manages to find the json file but gives the error:
`Either the entered path does not exist, Path searched: /dls/e01/data/2025/cm40602-4/raw/test_MD, or a different error has occured`
See here:
<img width="1151" height="513" alt="image" src="https://github.com/user-attachments/assets/7b90e727-af4f-43bf-b371-e08bedef18d7" />